spec/form_models/jobseekers/job_application/professional_status_form_spec.rb

```diff
@@ -42,7 +41,6 @@
       let(:attributes) { { professional_status_section_completed: true } }
 
       it { is_expected.to validate_inclusion_of(:qualified_teacher_status).in_array(%w[yes no on_track]) }
-      it { is_expected.to validate_inclusion_of(:statutory_induction_complete).in_array(%w[yes no]) }
```

Comment:
Aren't we missing the validation test over `is_statutory_induction_complete` ?

Reply:
see above - validations on a boolean